ddtrace/ddtrace.go

```diff
@@ -88,6 +88,9 @@ type FinishConfig struct {
 	// Error holds an optional error that should be set on the span before
 	// finishing.
 	Error error
+
+	// Disable backtrace generation in case of error.
```

Comment:
In Go, we have to start every documentation comment with the name of the identifier, please use the following:
```suggestion
	// NoDebugStack will prevent any set errors from generating an attached stack trace tag.
```